paddle/phi/kernels/fusion/gpu/fused_rope_utils.h

```diff
@@ -115,58 +257,86 @@ __global__ void VectorizedFusedRopeWithRotateEveryTwoKernel(
   int64_t size = batch_size * seq_len * num_heads * head_dim;
   MPType sin_value[VecSize];
   MPType cos_value[VecSize];
+
+  if (static_cast<int>(rotary_emb_base) == kDefaultRotaryBase) {
```

Comment:
Why cast to `int`? What if `rotary_emb_base = 10000.1`?

Should be something like `if (rotary_emb_base == static_cast<decltype(rotary_emb_base)>(kDefaultRotaryBase)`.

Reply:
Done